KayKnows-frontend/src/app/app.component.css

```diff
@@ -0,0 +1,16 @@
+.header {
+    width: 100%;
+    background-color: #FFFFFF;
+    border-bottom: 2px solid lightgray;
+}
+
+.header img {
+    height: 50px;
+    margin: 10px 10px 5px 10px;
+}
+
+.name {
+    float: right;
+    margin: 20px 15px 5px 15px;
+    color: #202D52;
+}
```

Comment:
Picky but add a new line at the end of the file. I think that is just how things should be.